instrumentation/servlet/servlet-3.0/src/test/groovy/AbstractServlet3Test.groovy

```diff
@@ -89,7 +89,7 @@ abstract class AbstractServlet3Test<SERVER, CONTEXT> extends HttpServerTest<SERV
         "${SemanticAttributes.HTTP_METHOD.key()}" method
         "${SemanticAttributes.HTTP_STATUS_CODE.key()}" endpoint.status
         // exception bodies are not yet recorded
-        "${SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH.key()}" responseContentLength
+        "${SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH.key()}" { it == responseContentLength || it == 0 || endpoint == EXCEPTION }
```

Comment:
This is like supressing the assertion isn't it? In the success cases, it seemed to be `responseContentLength == 7` and `it == 0` and AFAICT that was because the response was mysteriously prepared before the counting